pom.xml

```diff
@@ -77,7 +77,7 @@
             <!-- JUnit 5 requires Surefire version 2.22.0 or higher -->
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>3.5.3</version>
+                <version>2.22.2</version>
```

Comment:
Why was this downgraded @piotrooo? This now causes

```
org.junit.platform.commons.JUnitException: OutputDirectoryProvider not available; probably due to unaligned versions of the junit-platform-engine and junit-platform-launcher jars on the classpath/module path.
```

for me. I'll upgrade it again as part of my PR.